client/editSchedule.js

```diff
@@ -110,10 +129,10 @@ setupCompetitionCalendar = function(template, $calendarDiv, $editModal) {
         $editModal.modal('show');
       },
       eventDrop: function(calEvent, delta, revertFunc, jsEvent, ui, view) { // Existing entry dragged
-        eventChanged(calEvent);
+        eventChanged(calEvent, revertFunc);
```

Comment:
I might actually try to add that to SimpleSchema, should I find a hidden energy resource somewhere.